### DISCUSSION TITLE
Image preview doesn't work

### DISCUSSION THREAD

**Discussion by JerryVDP** [coder/code-server#4893] [General] [answered]
Hi,

I'm unable to preview images when I click on them in file explorer (sidebar).

Does anybody know why and how to fix it?

![image](https://user-images.githubusercontent.com/99589825/154836386-08ca1b51-1cb9-4431-80c3-50e73b4c043e.png)

Thanks!

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Can you try sharing the DevTools console logs? I think it's HTTPS or Webview that's why.

**Reply by JerryVDP** [reply]
You're correct, it fails to register the service worker because I'm running code via HTTP and not HTTPS.
For anyone looking to solve this problem without configuring HTTPS, you can add your code host to the insecure list of your chrome-based browser:
![image](https://user-images.githubusercontent.com/99589825/154838505-deea98bf-9220-4dae-8d8f-7561939eed7e.png)

**Comment by lg-nineoneninethreenine** [comment]
Solved by @JerryVDP:
> You're correct, it fails to register the service worker because I'm running code via HTTP and not HTTPS.
> For anyone looking to solve this problem without configuring HTTPS, you can add your code host to the insecure list of your chrome-based browser:
> ![image](https://user-images.githubusercontent.com/99589825/154838505-deea98bf-9220-4dae-8d8f-7561939eed7e.png)
> 
> > Link: https://github.com/coder/code-server/discussions/4893#discussioncomment-2213861

It is Webview.

**Reply by lg-nineoneninethreenine** [reply]
Just giving the right credits.

**Reply by jsjoeio** [reply]
Teamwork makes the dreamwork!
